lib/Frontend/CompilerInvocation.cpp

```diff
@@ -886,6 +899,41 @@ static bool ParseSearchPathArgs(SearchPathOptions &Opts,
   return false;
 }
 
+static void ValidateLocatizationArgs(ArgList &Args, DiagnosticEngine &Diags) {
```

Comment:
The uppercase in method name here is intentional?

Reply:
Yes, all of the methods here have an upper case first character. Should I change it?

Reply:
Ah nevermind just saw the pattern in the call :)